### PR TITLE
MAINT-36593 :Can’t download a file with name containing %

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

File: core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/DownloadConnector.java [hunk 1]
```diff
@@ -66,7 +69,11 @@ public Response download(@PathParam("workspace") String workspace,
         fileName = node.getProperty("exo:title").getString();
       }
       // decode the fileName in case the fileName is already encoded, for the old uploaded files.
-      fileName = URLDecoder.decode(fileName, "UTF-8");
+      try {
+        fileName = URLDecoder.decode(fileName, "UTF-8");
+      }catch (Exception e){
+        LOG.info("The fileName is already decoded");
```
Comment (ahamdi) [inline core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/DownloadConnector.java:75]: Change it to debug, no need to show this information.

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/DownloadConnector.java (hunk 1 above)
Comment (Rdidmohamedelhadi) [reply core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/DownloadConnector.java:75]: ok

Comment (hbenali) [reply core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/DownloadConnector.java:75]: Wouldn't worth also to log the stack trace on the debug module? WDYT @ahamdi

Comment (ahamdi) [reply core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/DownloadConnector.java]: No need for a stack trace as we will ignore the exception which is caused by the fact that the filename is already decoded.

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**